docs/pages/desktop-access/rbac.mdx

```diff
@@ -0,0 +1,138 @@
+    # Label selectors for desktops this role has access to.
+    windows_desktop_labels:
+      environment: ["dev", "stage"]
+
+    # Windows user accounts this role can connect as.
+    windows_desktop_logins: ["Administrator", "{{internal.windows_logins}}"]
+```
+
+<Admonition type="warning" title="Active Directory Configuration">
+Teleport's RBAC system is not a replacement for proper Active Directory
+administration. Teleport-issued Windows certificates are valid for a small
+amount of time, but they do apply to the entire domain. Proper care should be
+taken to ensure that each Teleport user's roles reflect only the necesary
+Windows logins, and that these Windows users are properly secured.
+</Admonition>
+
+## Labeling
+
+Both `allow` and `deny` rules support `windows_desktop_labels` selectors. These
+selectors are matched against the labels set on the desktop. It is possible to
+use wildcards (`"*"`) to match all desktop labels.
+
+Windows desktops acquire labels in two ways:
+
+1. The `host_labels` rules defined in the `windows_desktop_service` section of
+   your Teleport configuration file.
+2. Automatic `teleport.dev/` labels applied by Teleport (for desktops discovered
+   via LDAP only)
+
+For example, the following `host_labels` configuration would apply the
+`environment: dev` label to a Windows desktop named `test.dev.example.com`
+and the `environment: prod` label to `desktop.prod.example.com`:
+
+```yaml
+  host_labels:
+  - match: '^.*\.dev\.example\.com$'
+    labels:
+      environment: dev
+  - match: '^.*\.prod\.example\.com$'
+    labels:
+      environment: prod
+```
+
+For desktops discovered via LDAP, Teleport applies the following labels automatically:
+
+| Label | LDAP Attribute | Example |
+| ----- | -------------- | ------- |
+| `teleport.dev/computer_name` | `name` | `WIN-I5G06B8RT33`
+| `teleport.dev/dns_host_name` | [`dNSHostName`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-dnshostname) | `WIN-I5G06B8RT33.example.com`
+| `teleport.dev/os` | [`operatingSystem`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-operatingsystem) | `Windows Server 2012`
+| `teleport.dev/os_version`| [`osVersion`](https://docs.microsoft.com/en-us/windows/win32/adschema/a-operatingsystemversion) | `4.0`
+| `teleport.dev/windows_domain`| Sourced from config | `example.com`
+| `teleport.dev/is_domain_controller` | `primaryGroupID` | `true`
+
+## Logins
+
+The `windows_desktop_logins` role setting lists the Windows user accounts that
+the role permits access to. For local users, the `{{internal.windows_logins}}`
+variable can be used as a placeholder for the user's `windows_logins` trait. The
+`windows_logins` trait can be specified when the user is created with
+`tctl users add alice --windows-logins=Administrator,DBUser`.
+
+New clusters automatically populate the preset `access` role with the following:
+
+```yaml
+allow:
+  windows_desktop_logins: ["{{internal.windows_logins}}"]
+```
+
+## Clipboard Access
```

Comment:
Maybe in this section would be a good place to add an admonition specifying which browsers support clipboard sharing.